Common/src/CConfig.cpp

```diff
@@ -9221,7 +9233,7 @@ su2double CConfig::GetIsothermal_Temperature(const string& val_marker) const {
     if (Marker_Isothermal[iMarker_Isothermal] == val_marker)
       return Isothermal_Temperature[iMarker_Isothermal];
 
-  return Isothermal_Temperature[0];
+  return Temperature_FreeStream;
```

Comment:
Motivation?

Reply:
It was a change I implemented to cope with CHT boundaries with the flamelet solver, but I implemented a work-around later on and forgot to change this back.